elasticsearch_dsl/search.py

```diff
@@ -345,6 +345,7 @@ def __getitem__(self, n):
             # stop not given.
             s._extra['from'] = n.start or 0
             s._extra['size'] = n.stop - (n.start or 0) if n.stop is not None else 10
+            s._extra['size'] = max(s._extra['size'], 0)
```

Comment:
Can we move the `max(X, 0)` call on the same line as where `_extra['size']` is calculated above?